build-support/bin/BUILD

```diff
@@ -16,7 +16,7 @@ python_sources(
 
 python_tests(
     name="py_tests",
-    overrides={"reversion_test.py": {"timeout": 90, "dependencies": ["3rdparty/python#pex"]}}
+    overrides={"reversion_test.py": {"timeout": 90, "dependencies": ["3rdparty/python#pex"]}},
```

Comment:
I'm not sure how #16029 went green on main: https://github.com/pantsbuild/pants/actions/runs/2599496401

This is tiny; so I've forked off this same `./pants update-buildfiles` fix on main in #16049.